agent/php_execute.c

```diff
@@ -874,6 +880,30 @@ static void nr_execute_handle_library(const char* filename TSRMLS_DC) {
   nr_free(filename_lower);
 }
 
+static void nr_execute_handle_logging_framework(const char* filename TSRMLS_DC) {
+  char* filename_lower = nr_string_to_lowercase(filename);
+  size_t i;
+
+  for (i = 0; i < num_logging_frameworks; i++) {
+    if (nr_stridx(filename_lower, logging_frameworks[i].file_to_check) >= 0) {
+      char* metname = nr_formatf("Supportability/Logging/PHP/%s/enabled",
```

Comment:
I think this metric depends on if the log forward is enabled (and max log event > 0) and not just detection?

Reply:
According to the spec, `Supportability/Logging/{language}/{framework}/{enabled|disabled}` - enabled MUST be implemented, disabled MAY be implemented, but does not depend on the state of individual features. From the spec: 

> This metric does not take the enabled/disabled state of individual features (i.e. metrics, forwarding) into account.

Reply:
Should it depend on the global `newrelic.application_logging.enabled`?

Reply:
I agree, I think that metric should depend on the global newrelic.application_logging.enabled.

We could use Supportability/library/Monolog/detected for library detected.
And we could use Supportability/Logging/Php/Monolog/enabled if we detect  a logging library it AND if logging is enabled

Reply:
@zsistla @mfulb @bduranleau-nr This has been addressed in aaa733c9d8335c06031cbe0370eb5d8fa367d8d0 and refactored in 03b1abd51e6853f5e2c5ebef5053420ac10b2d82.